app/pages/sections/catalogue-solutions/select-price/ui-tests/general.ui.test.js

```diff
@@ -0,0 +1,190 @@
+  await pageSetup(t, false, false);
+  nock('http://identity-server')
+    .get('/login')
+    .reply(200);
+
+  await t.navigateTo(pageUrl);
+
+  await t
+    .expect(getLocation()).eql('http://identity-server/login');
+});
+
+test('should render Catalogue-solutions price page', async (t) => {
+  await pageSetup(t, true);
+
+  await t.navigateTo(pageUrl);
+  const page = Selector('[data-test-id="solutions-price-page"]');
+
+  await t
+    .expect(page.exists).ok();
+});
+
+test('should navigate to /organisation/order-id/catalogue-solutions/select-solution when click on backlink', async (t) => {
+  await pageSetup(t, true, true);
+  nock(orderApiUrl)
+    .get('/api/v1/orders/order-id/sections/supplier')
+    .reply(200, { supplierId: 'supp-1' });
+  await t.navigateTo(pageUrl);
+
+  const goBackLink = Selector('[data-test-id="go-back-link"] a');
+
+  await t
+    .expect(goBackLink.exists).ok()
+    .click(goBackLink)
+    .expect(getLocation()).eql('http://localhost:1234/order/organisation/order-id/catalogue-solutions/select-solution');
+});
+
+test('should render the title', async (t) => {
+  await pageSetup(t, true, true);
+  await t.navigateTo(pageUrl);
+
+  const title = Selector('h1[data-test-id="solutions-price-page-title"]');
+
+  await t
+    .expect(title.exists).ok()
+    .expect(await extractInnerText(title)).eql(`${content.title} order-id`);
+});
+
+test('should render the description', async (t) => {
+  await pageSetup(t, true, true);
+  await t.navigateTo(pageUrl);
+
+  const description = Selector('h2[data-test-id="solutions-price-page-description"]');
+
+  await t
+    .expect(description.exists).ok()
+    .expect(await extractInnerText(description)).eql(content.description);
+});
+
+test('should render a selectSolutionPrice question as radio button options', async (t) => {
+  await pageSetup(t, true, true);
+  await t.navigateTo(pageUrl);
+
+  const selectSolutionPriceRadioOptions = Selector('[data-test-id="question-selectSolutionPrice"]');
+
+  await t
+    .expect(selectSolutionPriceRadioOptions.exists).ok()
+    .expect(await extractInnerText(selectSolutionPriceRadioOptions.find('legend'))).eql(content.questions[0].mainAdvice)
+    .expect(selectSolutionPriceRadioOptions.find('input').count).eql(3)
+
+    .expect(selectSolutionPriceRadioOptions.find('input').nth(0).getAttribute('value')).eql('£1.64 per patient per year')
+    .expect(await extractInnerText(selectSolutionPriceRadioOptions.find('label').nth(0))).eql('£1.64 per patient per year')
+
+    .expect(selectSolutionPriceRadioOptions.find('input').nth(1).getAttribute('value')).eql('£525.052 per licence ')
+    .expect(await extractInnerText(selectSolutionPriceRadioOptions.find('label').nth(1))).eql('£525.052 per licence')
+    .expect(selectSolutionPriceRadioOptions.find('input').nth(2).getAttribute('value')).eql('<div>1 - 10 consultations £700 per consultation per month</div><div>11+ consultations £400 per consultation per month</div>')
```

Comment:
a blank line please after line 176 just to separate out each radioButton assertion